### ISSUE TITLE
Video playback stops if started in immersive mode

### ISSUE THREAD

**Issue by elima** [Igalia/wolvic#141]
# Configuration

Wolvic version: 0.9.7
Hardware: Oculus Quest 2

## Steps to Reproduce
1. Open a 360 video in e.g, Youtube <https://www.youtube.com/c/AirPanoVR/videos>
2. Click the fullscreen button before video starts to reproduce.

## Current Behavior
Scene is all black, there is no sound, playback doesn't progress, controls appear disabled.

## Expected Behavior
Video starts playing in immersive mode, with audio and all controls enabled.

**Comment by svillar** [comment]
I've debugged this a lot and I'm almost sure it's a Gecko issue. I reviewed our MediaSession code and also our YouTube extension. None of them seem to be the culprit.

What's more after trying a bit with the latest FirefoxReality release, I found that the bug was already present there. It's just that videos are usually loaded faster so it's a bit more difficult to reproduce, but the same bug is already there.
